rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly"
+components = ["rustc", "rustfmt"]
```

Comment:
What are the ramifications of spelling out the covered components.  Should we bring the other `rust-toolchain` files in the repo in line with this?

Reply:
I just pushed c536c095d3d28b5c83a044b4f398d76b10e019fc which specifies `profile = "default"` instead, in line with https://rust-lang.github.io/rustup/concepts/profiles.html.

I'll change the crucible rust-toolchain file to match this too.

Reply:
Squash that commit down into this one? No need to push both commits.

Reply:
Done